resources/aliases

```diff
@@ -126,3 +126,58 @@ function share() {
 function flip() {
     sudo bash /vagrant/scripts/flip-webserver.sh
 }
+
+function __has_pv() {
```

Comment:
I wasn't sure of the best way to handle this. I underscored these helpers because they're only meant to be used by the import/export functions. I wish Bash had classes and access control.